pytest_mypy_plugins/item.py

```diff
@@ -310,9 +310,6 @@ def prepare_mypy_cmd_options(self, execution_path: Path) -> List[str]:
         if not self.disable_cache:
             mypy_cmd_options.extend(["--cache-dir", self.incremental_cache_dir])
 
-        python_version = ".".join([str(part) for part in sys.version_info[:2]])
```

Comment:
I think that this might be a better option, because we don't interfer with mypy options 🤔 

Which one do you like best?

Reply:
Yeah, I think this makes sense. I think iIt's nice to be able to have parity between running `mypy` and the invocation that happens within pytest-mypy-plugins :)